en/troubleshoot-faq.md

```diff
@@ -132,6 +131,129 @@ access it, there are a few things to try:
    MAAS web interface has been installed in the right place. There should
    be a file present called `/usr/share/maas/maas/urls.py`.
 
+
+## Backdoor image login
+
+By design, ephemeral images are not built to be edited or tampered with.
+
+However, if you find yourself with no other way to access node, a Linux-based
+ephemeral image can be modified to enable a *backdoor* that adds or resets a
+user's password, much like you can with any standard version of Linux.
+
+As images are constantly updated and refreshed, the backdoor will only be
+temporary, but it should help you login to see what may be going wrong with
+your node.
+
+### Extract the cloud image
+
+First, download the cloud image that corresponds to the architecture of your
+node. The *Images* page of the web UI lists the images currently being cached
+by MAAS:
+
+![web UI image list][img__webui-images]
+
+Images can be downloaded from [https://cloud-images.ubuntu.com/daily/server](https://cloud-images.ubuntu.com/daily/server/).
+
+For example:
+
+```bash
+wget https://cloud-images.ubuntu.com/daily/server/xenial/current/xenial-server-cloudimg-amd64-root.tar.gz
+```
+
+With the image downloaded, extract its contents so that the *shadow* password
+file can be edited:
+
+```bash
+mkdir xenial
+sudo tar -C xenial -xpSf xenial-server-cloudimg-amd64-root.tar.gz --numeric-owner --xattrs "--xattrs-include=*"
+```
+
+!!! Note:
+    `sudo` is required when extracting the image filesystem and when making
+    changes to the files extracted from the image filesystem.
+
+### Generate password hash
+
+Now generate a hashed password. Use the following Python 3 command, replacing
+**ubuntu** with the password you wish to use:
+
+```bash
+python3 -c 'import crypt; print(crypt.crypt("ubuntu", crypt.mksalt(crypt.METHOD_SHA512)))'
+```
+
+Output from the previous command looks like the following:
+
+```no-highlight
+$6$AaHblHl5KGrWBmPV$20ssynyY0EhcT9AwZgA2sTdYt4Bvd97bX7PjeyqVLKun2Hk3NBa8r7efM2duK7pi2dlnd5tG76I0dTUvjb6hx0
+```
+
+Open the `etc/shadow` file extracted from the image with your favourite text editor:
+
+```bash
+sudo vim xenial/etc/shadow
```

Comment:
I'm not sure if we ever mention an actual editor in other parts of the docs. Seems we just say 'edit a file'. Just a comment.